src/06-loops.js

```diff
@@ -137,7 +188,16 @@ function firstIndexFound(array, target) {}
  *  lastIndexFound([ "left", "right", "left" ], "up");
  *  //> -1
  */
-function lastIndexFound(array, target) {}
+function lastIndexFound(array, target) {
+  let foundIndex = -1;
+  for(let i = array.length; i >= 0; i--){
+    if (array[i] == target){
+      foundIndex = i;
+    }
+    if (foundIndex != -1) break; 
```

Comment:
Here we can move our `break` statement inside of our `if` block.  Since we are iterating backwards and have found an match there is nothing wrong with simply breaking out of loop early as soon as we reassign `foundIndex`